andela/mag-ah-frontend
# #159965321 Setup react-router version 4

#### What does this PR do?
- Set up react-router version 4

#### Description of the task to be completed?
- Set up react router version 4
- Create a components and views file structure where view pages are in the views folder but individual components are in the components folder.
#### How should this be manually tested?
$ git clone https://github.com/andela/mag-ah-frontend.git
$ git checkout ch-setup-react-routerV4-159965261
$ cd mag-ah-frontend
$ npm install
$ npm start
navigate to `http://localhost:3000/`
You should see the following page:
<img width="1095" alt="screen shot 2018-10-03 at 14 35 03" src="https://user-images.githubusercontent.com/20840601/46408100-a28c1780-c719-11e8-9c82-e75e5941f099.png">

navigate to `http://localhost:3000/login`
You should see the following page:
<img width="1121" alt="screen shot 2018-10-03 at 14 35 13" src="https://user-images.githubusercontent.com/20840601/46408129-c18aa980-c719-11e8-8430-e80313caecdd.png">



#### What are the relevant pivotal tracker stories?
[#159965261](https://www.pivotaltracker.com/story/show/159965261)
